### ISSUE TITLE
[Enhancement] Sheet backdrop is shown over the navigation bar  

### ISSUE THREAD

**Issue by Sebosek** [DIPSAS/DIPS.Xamarin.UI#296]
## Summary
As is defined in spec https://github.com/DIPSAS/DIPS.Xamarin.UI/issues/94, it would be fine to have an option to cover the navigation bar with a sheet backdrop to have a consistent feel from the application. Without that the content of the Content page and its Navigation bar feel disconnected.

## API Changes

Honestly, not sure if it could be done by setting sheet prop like `IsNavigationIncluded="True"` or it would require more XAML/platform-specific magic. But, I believe a boolean value that would include a navigation bar under the backdrop should be enough.

## Intended Use Case
In a nutshell, better UX.
In FAQ there is a tip to show/hide navigation based on the sheet visibility, but it causes "bumping" of UI in the background and the overall user experience is not good.

Or is there a way, how to cover the navigation bar by the backdrop, when a sheet is open?

**Comment by haavamoa** [comment]
Hi and thank you for this issue. This is indeed something we want to support as it will lead to better UX. Unfortunately our sheet component does not "know" about the navigation bar or the page it is attached to, so it can not intercept it to make sure the backdrop covers the navigation bar. I am not sure if its easy to do even if it did "know" about the page and nav bar. 

We do want our sheet to feel more "native" without us having to maintain the XAML version we have created, which means we will at some point transition to the Android and iOS APIs. Reason this is not the case is because we found it hard to use these APIs together with XAML and the Xamarin way if I am not mistaken. I am unable to give you a expected date to When we will be give the native APIs another try. Reason is that we have not gotten any negative feedback from the users of our apps when it comes to our sheet component, so its not a high enough priority as of right now.

Sorry for the late reply 😅 Have a nice day. 👍

**Comment by haavamoa** [comment]
This is fixed with our new MAUI library, we wont fix it in this Xamarin version:

https://github.com/DIPSAS/DIPS.Mobile.UI
